### PR TITLE
docs: explain how to test single block/superblock

### LINKED ISSUE #49403: Update test-challenges/docs for testing specific blocks
https://github.com/freeCodeCamp/freeCodeCamp/pull/49293 stops Windows users from being able to restrict tests to specific blocks or superblocks.

Ideally the all users should be able to run the scripts the same way (maybe just via FCC_BLOCK/FCC_SUPERBLOCK In .env), but regardless the docs will need updating.

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes https://github.com/freeCodeCamp/freeCodeCamp/issues/49403

### CODE REVIEW

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]

File: docs/how-to-work-on-coding-challenges.md [hunk 1]
```diff
@@ -460,7 +460,7 @@ Before you [create a pull request](how-to-open-a-pull-request.md) for your chang
 pnpm run test:curriculum
 ```
 
-2. You can also test a block or a superblock of challenges with these commands
+2. You can also test a block or a superblock of challenges by passing the environment variable `FCC_BLOCK` or `FCC_SUPERBLOCK` respectively. For example, on 'nix systems:
```
[the remark this answers is not in the document]
Comment (ojeytonwilliams) [reply docs/how-to-work-on-coding-challenges.md:463]: I meant unix-based systems in general, but we could just say "on Linux".

[the remark this answers is not in the document]
Comment (Sembauke) [reply docs/how-to-work-on-coding-challenges.md:463]:
```suggestion
2. To test a block or a superblock of challenges, you can pass the environment variables `FCC_BLOCK` or `FCC_SUPERBLOCK`, respectively. For instance, on Unix-based systems:
```

**Comment by Sembauke** [comment]
@ojeytonwilliams Hey how's this going :)

**Comment by raisedadead** [comment]
Hi, 

We are closing old issues/pull requests that we think are stale. If you still need help or are fixing this, please open a new issue/pull request.

Thanks for your understanding. 

Happy contributing.

**Pull request closed without merging**